dev/AppNotifications/AppNotifications.idl

```diff
@@ -111,6 +111,29 @@ namespace Microsoft.Windows.AppNotifications
 
         // Gets or sets whether a Notification's pop-up UI is displayed on the user's screen.
         Boolean SuppressDisplay;
+
+        // Gets or sets the Notification Device Data
+        AppNotificationDevicesData DevicesData;
+    }
+
+    // The Notification Device Data
+    [contract(AppNotificationsContract, 2)]
+    runtimeclass AppNotificationDevicesData
+    {
+        // Initializes a new Instance of NotificationDevicesData
+        AppNotificationDevicesData();
+
+        // Checks if Video or Audio Calling is supported
+        static Boolean IsVideoOrAudioCallingSupported();
+
+        // Gets or sets the Camera Device Id
+        String CameraDeviceId{ get; set; };
+
+        // Gets or sets the Microphone Device Id
+        String MicrophoneDeviceId{ get; set; };
```

Comment:
For read/write properties, use the compact IDL syntax - `String AudioInputDeviceId;`

Reply:
changed to 
        // Gets or sets the Video Device Id
        String VideoDeviceId;

        // Gets or sets the Microphone Device Id
        String AudioInputDeviceId;

        // Gets or sets the Speaker Device Id
        String AudioOutputDeviceId;